.eslintrc.js

```diff
@@ -1291,8 +1291,8 @@ module.exports = {
       plugins: ['react', '@typescript-eslint'],
       files: ['x-pack/plugins/osquery/**/*.{js,mjs,ts,tsx}'],
       rules: {
-        'arrow-body-style': ['error', 'as-needed'],
-        'prefer-arrow-callback': 'error',
+        // 'arrow-body-style': ['error', 'as-needed'],
```

Comment:
Why did this need disabled?

Reply:
those rules are only for `osquery` that I'm working on, so just temporarily disabled them to limit the number of changes in this PR